backend/tests/unit/repositories/test_history_repository.py

```
@@ -867,13 +908,36 @@ def test_get_session_timeline_database_error(self, repository_with_session_error
-    
+
+    @pytest.mark.unit
+    def test_update_alert_session_database_error(self, repository_with_session_error, sample_alert_session):
+        """Test update alert session with database error."""
+        result = repository_with_session_error.update_alert_session(sample_alert_session)
+        assert result == False
+
+    @pytest.mark.unit
+    def test_get_llm_interactions_database_error(self, repository_with_session_error):
+        """Test getting LLM interactions with database error."""
+        with pytest.raises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)